### PR TITLE
[QA] Signal Stability acceptance tests

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers
@flopezag @geonexus 

#### Description
Signal Stability test features. This PR solves *CLAUDIA-4738*

#### Testing
Locally.

### CODE REVIEW

**Comment by flopezag** [comment]
The case 2/3 and 3/3 are bugs like they are described, the time means the different between the first time value and the last time values in the windows have to be less than the limit time size.

In the case of those tests, due to we wait 15 seconds (gt the 10 seconds defined by default), this time difference is for sure gt the allowed time due to the second and third values are yet gt 10 seconds.

**Comment by flopezag** [comment]
I suggest to add in the description of the Scenario that we have preconfigured a time value for the maximum allowed difference time in the window.

**Comment by geonexus** [comment]
LGTM

File: tests/acceptance/features/component/signal_stability.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  I want to have a temporal window
+  in order to reset the grouping when the period between facts is higher than the configured one.
+
+
+  @happy_path @skip @bug @CLAUDIA-5528
+  Scenario: Facts grouping by window size is reset when the time between facts is higher than the configured one (10 seconds by default) (1/3).
+    Given the configured tenant-id is registered in CLOTO component
+    And   RabbitMQ consumer is looking into the configured message bus
+    And   the context notification has default context elements
+    And   window size is set to "3"
```
Comment (flopezag) [inline tests/acceptance/features/component/signal_stability.feature]: You forget to put explicitly the default time size of the windows in the Ands of the 3 scenarios.

Comment (jframos) [inline tests/acceptance/features/component/signal_stability.feature]: I put it in the Scenario description. I think that is better place than as Step. 
(I cannot split the description in more lines).

Comment (flopezag) [inline tests/acceptance/features/component/signal_stability.feature]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**